### DISCUSSION TITLE
My account has been flagged without reason

### DISCUSSION THREAD

**Discussion by cabrata** [community/community#124379] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

hello, my main github with the username @caliphdev has been flagged for unknown reasons, and I have created a support ticket but have not been replied to since last week, please help

**Comment by ebndev** [comment]
Hi @caliph91,

This community is managed by GitHub employees, but it is not a support forum. You have done everything necessary on your end by opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
